cmd/flightctl-server/main.go

```diff
@@ -62,12 +63,17 @@ func main() {
 		log.Fatalf("running initial migration: %v", err)
 	}
 
+	log.Println("Initializing async jobs")
+	taskManager := tasks.Init(log, store)
+	taskManager.Start()
+	defer taskManager.Stop()
+
```

Comment:
nit: would love this to live in Server if possible. It simplifies tests to not need to create the taskManager since server already has its deps.